requirements.txt

```diff
@@ -1,12 +1,44 @@
-Flask==0.10.1
-Jinja2==2.7.3
-MarkupSafe==0.23
-Werkzeug==0.9.6
-gnureadline==6.3.3
-itsdangerous==0.24
-rauth==0.7.0
-requests==2.3.0
-wsgiref==0.1.2
-
-gunicorn==18.0
-Flask-SSLify==0.1.4
+aniso8601==7.0.0
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-29217   
**Check Name:** python-jwt: Key confusion through non-blocklisted public key formats   
**Severity:** HIGH   
**Fixed Version:** 2.4.0   
**Description:** PyJWT is a Python implementation of RFC 7519. PyJWT supports multiple different JWT signing algorithms. With JWT, an attacker submitting the JWT token can choose the used signing algorithm. The PyJWT library requires that the application chooses what algorithms are supported. The application can specify `jwt.algorithms.get_default_algorithms()` to get support for all algorithms, or specify a single algorithm. The issue is not that big as `algorithms=jwt.algorithms.get_default_algorithms()` has to be used. Users should upgrade to v2.4.0 to receive a patch for this issue. As a workaround, always be explicit with the algorithms that are accepted and expected when decoding.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-29217